openedx/openedx-platform
# feat: paste tags when pasting xblocks with tag data [FC-0049]

## Description
This PR changes the paste XBlock feature to apply tags to the destination block.

## More information
- Closes https://github.com/openedx/modular-learning/issues/180

Depends on:
- https://github.com/openedx/openedx-learning/pull/164
- https://github.com/openedx/edx-ora2/pull/2184

## Testing instructions
- Make sure the tests cover the various cases.
- Start you localdev stack on this branch
- Make sure you have Taxonomy/Tags data setup locally: https://github.com/open-craft/taxonomy-sample-data/
- Once you have a course with a bunch of units and blocks with tags applied, try copying units and various types of blocks
- Paste the blocks to another place and check if the tags are applied correctly. Note that if you paste a block from a course from one org to another, only the tags that are associated with the destination org will be applied.

___
Private ref: [FAL-3621](https://tasks.opencraft.com/browse/FAL-3621)
